ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productlist/v2/productlist/_cq_dialog/.content.xml

```diff
@@ -53,6 +58,35 @@
                         uncheckedValue="false"
                         value="true"
                         checked="{String}true"/>
+                    <xfItems
+                        granite:class="foundation-toggleable cmp-productcollection__editor-xf_settings"
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
+                        composite="{Boolean}true"
+                        fieldDescription="Allows insertion of Experience Fragments at specified positions"
+                        fieldLabel="Experince Fragment placeholders">
+                        <field
+                            granite:class="cmp-productcollection__editor-xf_setting"
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="granite/ui/components/coral/foundation/container"
+                            name="./fragments">
+                            <items jcr:primaryType="nt:unstructured">
+                                <xfLocation
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    fieldDescription="The name of this Experience Fragment location."
+                                    fieldLabel="Experience Fragment location name."
+                                    required="{Boolean}true"
+                                    name="fragmentLocation"/>
+                                <xfPosition
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    fieldDescription="The CSS class to be applied to the Experience Fragment container"
+                                    fieldLabel="Content CSS class"
+                                    name="fragmentCssClass"/>
```

Comment:
Editors usually should not care about CSS classes. These should be configured as options in the policy of the component and given here as a select. Same as for the style system.